pkg/agent/flowexporter/exporter/priorityqueue/priorityqueue.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2021 Antrea Authors
```

Comment:
that level of nesting doesn't seem necessary to me, maybe this file could be moved to `pkg/agent/flowexporter/exporter/priorityqueue.go` or `pkg/agent/flowexporter/priorityqueue/priorityqueue.go` instead. @srikartati

Reply:
I prefer the latter one, so that we can move ItemToExpire struct back into priority queue struct as previous, also without introducing import cycle.